WalletWasabi.Gui/CrashReport/Views/CrashReportWindow.xaml.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+using System.ComponentModel;
+using Avalonia;
+using Avalonia.Markup.Xaml;
+using AvalonStudio.Shell.Controls;
+using Splat;
+
+namespace WalletWasabi.Gui.CrashReport.Views
+{
+	public class CrashReportWindow : MetroWindow
```

Comment:
Shouldnt use metro window here, should use...`WasabiWindow`

Reply:
ACK.